src/main/java/bsh/ClassGenerator.java

```diff
@@ -73,8 +73,8 @@ public Object invokeSuperclassMethod(BshClassManager bcm, Object instance, Strin
     public static Class generateClassImpl(String name, Modifiers modifiers, Class[] interfaces, Class superClass, BSHBlock block, Type type, CallStack callstack, Interpreter interpreter) throws EvalError {
```

Comment:
I see this didn't add the wildcard type parameters for ie. `Class<?>` and `Constructor<?>` any reason why it can't?

Reply:
It can but I had this particular replacement disabled to limit the diff.